build-tarball.sh

```diff
@@ -29,6 +29,18 @@ cp $dir/recovery.img $wDir/partitions
 cp -r "$device_dir/ubuntu-overlay/*" $wDir/ || true
 
 mkdir -p $wDir/system/var/lib/lxc/android/
+
+## SPARSE FILE TRANSLATION
+# Needed with Halium-7.1 builds mostly
+isSparse=$(file -b0 $dir/system.img)
+if [[ $isSparse == "Android sparse image"* ]]; then
```

Comment:
I question the name of the variable isSparse. Maybe `systemFileType` would make more sense. `isSparse` implies its value is boolean.